src/Materials/Node/nodeMaterial.ts

```diff
@@ -1677,6 +1677,53 @@ export class NodeMaterial extends PushMaterial {
         this._mode = source.mode ?? NodeMaterialModes.Material;
     }
 
+    /**
+     * Load new blocks and frame from a frame serialization object
+     * @param source defines the JSON representation of the frame
+     * @param rootUrl defines the root URL to use to load textures and relative dependencies
+     */
+    public addFrameNodesFromeSerialization(source: any, rootUrl: string = "") {

```

Comment:
I would like to let the NOdeMaterial unchanged

Reply:
No code  should be added on bjs core

Reply:
moved to graphEditor